package.json

```diff
@@ -111,8 +113,8 @@
 		"typecheck": "tsc --noEmit",
 		"update-deps": "npx rimraf package-lock.json && npm run -s distclean && npm i && replace --silent 'http://' 'https://' . --recursive --include='package-lock.json' && touch -m node_modules",
 		"validate-fallback-es5": "npx eslint --parser-options=ecmaVersion:5 --no-eslintrc --no-ignore ./public/fallback/*.js",
-		"prewhybundled": "cross-env-shell CALYPSO_ENV=production EMIT_STATS=withreasons NODE_ARGS=--max_old_space_size=8192 npm run -s build-client",
-		"whybundled": "whybundled stats.json",
+		"prewhybundled": "cross-env-shell CALYPSO_ENV=production EMIT_STATS=withreasons npm run -s build-client-evergreen",
```

Comment:
I don't think we need to worry about x-platform support here, we can do something like the following:

```
[[ -f client/stats.json ]] && echo '`client/stats.json exists, skipping generation. To rebuild, remove the stats.json file.' || CALYPSO_ENV=production EMIT_STATS=withreasons npm run -s build-client-evergreen
```

Reply:
Do bear in mind that the stats file may already exist but not be the one we want, e.g. if `npm run analyze-bundles` was previously run. If we're going to do this, we should rename the file to `stats-with-reasons.json` to make it very clear this is the one we want.

Also, not sure how we should handle staleness. Presumably the developer will be able to tell that it took no time at all and thus it was an existing, possibly stale file?